src/rosdistro/manifest_provider/git.py

```diff
@@ -107,4 +116,4 @@ def _temp_git_clone(url, ref):
 
         yield base
     finally:
-        shutil.rmtree(base)
+        shutil.rmtree(base, onerror=onerror)
```

Comment:
The `onerror` handler is only necessary on Windows and therefore should only be added on that platform.